lib/datasource/artifactory/__snapshots__/index.spec.ts.snap

```diff
@@ -5,19 +5,19 @@ Object {
   "registryUrl": "https://jfrog.company.com/artifactory",
   "releases": Array [
     Object {
-      "releaseTimestamp": "2021-07-21T20:08:00.000Z",
```

Comment:
Revert this file

Reply:
oh, of course - that change was done unintentionally